docs/coco.md

```diff
@@ -48,6 +48,26 @@ coco_image.add_annotation(
   )
 )
 ```
+- add predictions to coco image:
+
+```python
+coco_image.add_prediction(
+  CocoPrediction(
+    score=0.864434,
+    bbox=[x_min, y_min, width, height],
+    category_id=0,
+    category_name='human'
+  )
+)
+coco_image.add_prediction(
+  CocoPrediction(
+    score=0.653424,
+    segmentation=[x_min, y_min, width, height],
```

Comment:
`segmentation` format is a bit different. Should be in the form of `[polygon1, polygon2, ...]` where polygons are in the form [x1, y1, x2, y2, x3, y3, ...]

It's too complex to show in an example, maybe we can directly delete the segmentation example in docs?